internal/cli/apis.go

```diff
@@ -276,6 +316,8 @@ auth0 apis update <id|audience> --name myapi`,
 
 	apiName.RegisterStringU(cmd, &inputs.Name, "")
 	apiScopes.RegisterStringSliceU(cmd, &inputs.Scopes, nil)
+	apiAllowOfflineAccess.RegisterBool(cmd, &inputs.AllowOfflineAccess, true)
```

Comment:
```suggestion
	apiAllowOfflineAccess.RegisterBool(cmd, &inputs.AllowOfflineAccess, false)
```
This one shouldn't be required. I did not include this on the card, my bad.